src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -148,15 +148,15 @@ export default class ControlledTable extends PureComponent<ControlledTableProps,
 
         // Adjust [fixed columns/fixed rows combo] to fixed rows height
         let trs = r0c1.querySelectorAll('tr');
-        r0c0.querySelectorAll('tr').forEach((tr, index) => {
+        Array.from(r0c0.querySelectorAll('tr')).forEach((tr, index) => {
```

Comment:
querySelectorAll returns an array-like object (NodeList), while Array.forEach existed in ES5 and was supported by IE 11, NodeList does not. Converting for use.